core/src/main/java/mx/yellowme/youst/core/hooks/BaseFragment.kt

```diff
@@ -0,0 +1,33 @@
+package mx.yellowme.youst.core.hooks
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+abstract class BaseFragment : Fragment() {
+
+    abstract val layoutId: Int
+
+    //region Lifecycle
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(layoutId, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        setup()
+    }
+
+    //endregion
+
+    abstract fun setup()
```

Comment:
What do u think on renaming this method to something like `afterViewCreated` or `onViewReady`